certbot-apache/MANIFEST.in

```diff
@@ -2,6 +2,6 @@ include LICENSE.txt
 include README.rst
 recursive-include docs *
 recursive-include certbot_apache/tests/testdata *
```

Comment:
Given what happened with the refactor `certbot` tests PR, I actually think moving them inside `_internal` is pretty much guaranteed to work, but moving them out may cause annoying bugs that take time to track down (see again previous PR). In which case if you're fine moving them into `_internal` I'd rather do that in this PR; this is basically why I decided to separate out the test-moving from the other internalizing in the first place.

Reply:
In the long run, I'd personally would still like `tests` to be moved up a level.

Absent that in the short term, I personally like `tests` being moved under `_internal` more than where they are now. I agree moving them up a level is harder, but after fixing up this `MANIFEST.in` line, imports, and any tests that break when the location of these files move, I'm not sure how much harder it'd really be.

I can make a call if you'd like, but I'm fine with whatever you'd like to do here.

Reply:
> after fixing up this MANIFEST.in line, imports, and any tests that break when the location of these files move

Yes, these are the hard parts, especially if the breaks are in things like the test farm tests. the problem is that moving them up takes quite thorough testing now so that we don't hit it at release later, which takes time, which is why I am fine doing it but would rather do it in a separate PR in case of unexpected bugs, so that at least this can get in.

Reply:
OK. My vote for what it's worth is to not to spend time moving them to `_internal` and try to move them up a level in a separate PR if we find the time. If we don't, I think it's OK.